.circleci/config.yml

```diff
@@ -31,12 +31,12 @@ aliases:
     restore_cache:
       name: cache | restore homebrew
       keys:
-        - v3-homebrew-{{ epoch }}
-        - v3-homebrew
+        - v4-homebrew-{{ .Branch }}-{{ epoch }}
```

Comment:
This is a temporary key for debugging to not share caches with other else branches.